scorecard/scorecard/scorecard.py

```diff
@@ -278,13 +397,25 @@ async def on_startup(app):
     gh_client = gidgethub.aiohttp.GitHubAPI(session, 'scorecard', oauth_token=token)
     app['gh_client'] = gh_client
 
-    await update_data(gh_client)
-    asyncio.ensure_future(poll(gh_client))
+    asana_token_file = os.environ.get('ASANA_TOKEN_PATH', '/asana/token.txt')
+    with open(asana_token_file, 'r') as f:
+        token = f.read().strip()
+    asana_session = ClientSession(headers={'Authorization': f'Bearer {token}'})
+    asana_client = AsanaClient(session=asana_session)
```

Comment:
I think the interface makes a bit more sense for the AsanaClient to take a `token` argument and if the token argument is `None`, it looks up the token as you do here. This will keep all the Asana-related stuff in one place, the AsanaClient. If we ever want to use Asana elsewhere, we can easily move that class.